app/test/frontend/golden/v2/web_landing_page.html

```diff
@@ -60,9 +60,8 @@ <h1 class="_visuallyhidden">Dart pub</h1>
     <main class="home-banner">
       <h2 class="_visuallyhidden">Dart package manager</h2><img class="logo" src="/static/v2/img/site-logo.svg" alt="Dart package manager">
       <p class="text">Find and use packages to build <a href="/experimental/flutter">Flutter</a>, <a href="/experimental/web">web</a> and <a href="/experimental/server">server</a> apps with <a target="_blank" href="https://www.dartlang.org">Dart</a>.</p>
-      <form class="search-bar" action="/experimental/search">
+      <form class="search-bar" action="/experimental/server/packages">
```

Comment:
shouldn't the web landing page point to web packages?

Reply:
good catch, bad input in test, fixing.